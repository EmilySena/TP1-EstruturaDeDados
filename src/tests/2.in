5
9845
33222
2368745
123
32000
0
0
-2
123
-3
-1
-2