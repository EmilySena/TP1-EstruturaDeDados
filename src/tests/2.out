nave 32000 em combate
nave 123 em combate
2368745
33222
9845
nave 123 avariada
123
nave 123 consertada
123
2368745
33222
9845
